src/OrchardCore.Modules/Orchard.Setup/Services/SetupService.cs

```diff
@@ -207,8 +208,10 @@ await scope
                     };
 
                     // Invoke modules to react to the setup event
-                    var eventBus = scope.ServiceProvider.GetService<IEventBus>();
-                    await eventBus.NotifyAsync<ISetupEventHandler>(x => x.Setup(
+                    var setupEventHandlers = scope.ServiceProvider.GetRequiredService<IEnumerable<ISetupEventHandler>>();
+                    var logger = scope.ServiceProvider.GetRequiredService<ILogger<SetupService>>();
```

Comment:
GetRequiredServices, then the Ienumerable is implied

Reply:
Right, I think I saw that after, will change.